developer-experience/readme.md

```diff
@@ -85,6 +85,12 @@ New team members are uniquely positioned to identify instances of undocumented [
 
 The following outline examples of several strategies that can be adopted to promote a positive developer experience. It is expected that each team should define what a positive dev experience means within the context of their project. Additionally, refine that over time via feedback mechanisms such as sprint and project retrospectives.
 
+### Observability

```

Comment:
I would perhaps move observability further down the list vs having it as the first in the strategies to have a positive DevEx.